### PR TITLE
add travis file

### PR DESCRIPTION
**Pull request by philbrookes**
add travis file

### CODE REVIEW

**Comment by mikenairn** [comment]
@philbrookes Looks like you added the binary file, was that intentional?

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+jobs:
+  include:
+    - stage: test
+      script:
+      - cd $HOME/gopath/src/github.com/integr8ly/$OPERATOR_NAME && make travis/setup test/unit
+    - stage: compile
+      script:
+      - mkdir -p build/_output/res
+      - cp -r res build/_output/res
```
Comment (mikenairn) [inline .travis.yml]: Whats happening here? Whatever this is should it not be done in the makefile?

Comment (philbrookes) [reply .travis.yml]: I tried to do it in the makefile and it wouldn't work, it was pretty confusing

Comment (mikenairn) [reply .travis.yml]: @odra Why is this needed? Is this to do with copying the template in the Dockerfile? Did we not change this to use packr instead?

Comment (philbrookes) [reply .travis.yml]: I've pushed it with the changes back in the Makefile, to see how it fails again

**Comment by philbrookes** [comment]
removed the binary file.

**Comment by philbrookes** [comment]
@odra  I've removed the line from the docker file that copies in the res, as I believe this is replaced with packr now, can you confirm that please?

**Comment by odra** [comment]
@philbrookes yes it uses packr to use the binary that line can be removed.

**Pull request merged**